SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -65,10 +68,26 @@ The tables below show where cross-server RPCs are executed based on where they w
 
 #### Invoking a cross-server RPC from a client-worker
 
-The call is not processed because this type of RPC is only for a server-worker instance to call.
+The SpatialOS Runtime does not process the call because this type of RPC is only available to a server-worker, not a client-worker.
 
+### Unresolved parameters in an RPC
+
+In some situations, a server-worker or client-worker might receive an RPC with a parameter that it cannot resolve: for example, an actor passed as a parameter may not be checked out by the receiver. When this happens, the GDK outputs a warning to the logs. However, there may be circumstances in which you don't want the GDK to output a warning because the behavior is acceptable; when this is the case, you can disable the warnings on a per-RPC basis, either using C++ or Blueprints.
```

Comment:
1. What does "checked out" mean here? The server-worker or client-worker that receives the RPC does not have interest in or does not have authority over the Actor? I am guessing authority?
(We have historically and ambiguously used "checked out" in SpatialOS to mean either a server-worker instance has active read access or CBI interest in (not the same thing) an entity's component. We no longer use this term as it's ambiguous.)
2. What's the receiver? The server-worker or client-worker receiving the RPC?
3. Output a warning to the logs - Is it really the Runtime that outputs the error? 
4. What does "behavior is acceptable" mean? Can we just cut that bit?

Made a guess...

```suggestion
In some situations, a server-worker or client-worker might receive an RPC with a parameter that it cannot resolve: for example, it may receive an Actor passed as a parameter which it does not have authority over. When this happens, the SpatialOS Runtime outputs a warning to the logs. However, there may be circumstances in which you don't want the Runtime to output a warning; when this is the case, you can disable the warnings on a per-RPC basis, either using C++ or Blueprints.
```